projects/default/controllers/ros/headersGenerator.py

```diff
@@ -72,13 +73,13 @@ def ros_md5sum(self, srv):
             accum = text_in
             for l in text.split('\n'):
                 l = l.split('#')[0].strip()  # strip comments
-                if "geometry_msgs/" in l or "std_msgs/" in l or "sensor_msgs/" in l or "Header" in l:
-                    # replace message by its corresponding md5
+                if "geometry_msgs/" in l or "std_msgs/" in l or "sensor_msgs/" in l or "Header" in l or 'webots_ros/' in l:
                     message = l.split(' ')[0]
-                    # the fact that it is an array is not releavant for the computation of the MD5
-                    message = message.replace("[]", "")
-                    if message in self.predifinedMD5:
-                        l = l.replace(message, self.predifinedMD5[message])
+                    # replace message by its corresponding md5
```

Comment:
Consistency with the following comment:
```suggestion
                    # replace message by its corresponding MD5
```